spid-spring-integration/src/test/java/it/italia/developers/spid/integration/config/IdpKeyManagerTest.java

```diff
@@ -0,0 +1,77 @@
+package it.italia.developers.spid.integration.config;
+
+import lombok.SneakyThrows;
+import org.assertj.core.api.SoftAssertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.opensaml.xml.security.credential.Credential;
+
+import java.security.cert.Certificate;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class IdpKeyManagerTest {
+
+    static IdpKeyManager idpKeyManager;
+
+    @SneakyThrows
+    @BeforeAll
+     static void startup(){
+        String entity ="test.idp.entity";
```

Comment:
Missing a space
```suggestion
        String entity = "test.idp.entity";
```

Reply:
This string is repeated a few times throughout the tests, consider extracting it to a constant.